morango/api/viewsets.py

```diff
@@ -71,19 +71,36 @@ def create(self, request):
         else:
             return response.Response(serialized_cert.errors, status=status.HTTP_400_BAD_REQUEST)
 
-    @decorators.action(methods=['post'], detail=False)
-    def push_certificate(self, request):
+    @decorators.action(methods=['post'], detail=False, authentication_classes=[], permission_classes=[])
```

Comment:
Removing permissions to allow anyone to push to a server that has `ALLOW_CERTIFICATE_PUSHING` set to `True`.